calico_kubernetes/calico_kubernetes.py

```diff
@@ -37,7 +44,9 @@ def __init__(self):
         self.profile_name = None
         self.namespace = None
         self.docker_id = None
+
         self._datastore_client = DatastoreClient()
+        self._ipam_client = IPAMClient()
```

Comment:
I'd keep `self.datstore_client`. We have a `docker_client` and `etcd_client` elsewhere, don't want to get confused